src/main/java/com/ericsson/eiffel/remrem/generate/cli/CLIOptions.java

```diff
@@ -140,5 +144,18 @@ public static boolean hasParsedOptions() {
         if (commandLine == null)
             return false;
         return commandLine.getOptions().length > 0;
+    }    
+    
+    public static void handleJarLocation(){
+        if (commandLine.hasOption("jp")) {
+            String jarDirectoryPath = commandLine.getOptionValue("jp");
```

Comment:
Why do we use the term Directory here? Why not stick with Path? Using different terminology for the same thing will only confuse.